protocol/scripts/governance/bridge_vesters_set_rewards.py

```diff
@@ -51,50 +63,45 @@
 DELAY_MSG_MODULE_ADDRESS="dydx1mkkvp26dngu6n8rmalaxyp3gwkjuzztq5zx6tr"
 DEPOSIT=f"10000{NINE_ZEROS}{NINE_ZEROS}{NATIVE_TOKEN_DENOM}" # 10,000 native tokens
 OUTPUT_FILE='proposal_bridge_vesters_set_rewards.json'
-REF_BLOCK_HEIGHT_FOR_ESTIMATE=1793496 
-REF_BLOCK_TIME_FOR_ESTIMATE="2023-11-23T17:10:19+00:00"
-
-# Helper function to estimate future block time.
-def estimate_block_height(current_block_height, current_block_timestamp, average_block_time, future_timestamp):
-    """
-    Estimates the block height at a future timestamp.
-
-    :param current_block_height: Current height of the blockchain (in blocks)
-    :param current_block_timestamp: UTC Timestamp of the current block (ISO 8601 format)
-    :param average_block_time: Average time it takes to mine a block (in seconds, can be a float)
-    :param future_timestamp: UTC Future timestamp for which block height is to be estimated (ISO 8601 format)
-    :return: Estimated block height at the future timestamp
-    """
+
+# Helper function to estimate number of blocks between two times.
+def estimate_blocks_between_timestamps(base_block_timestamp, average_block_time, future_timestamp):
     # Convert ISO 8601 timestamps to datetime objects
-    current_block_time = parser.isoparse(current_block_timestamp)
+    base_block_time = parser.isoparse(base_block_timestamp)
     future_time = parser.isoparse(future_timestamp)
 
     # Calculate the time difference in seconds
-    time_difference = (future_time - current_block_time).total_seconds()
+    time_difference = (future_time - base_block_time).total_seconds()
```

Comment:
should we add assertion that `future - base` is not negative?

Reply:
This is effectively checked below with
```
if delay_blocks_update_1 <= 0 or delay_blocks_update_2 <= 0:
    sys.exit(f"Estimated delay_blocks <= 0: {delay_blocks_update_1}, {delay_blocks_update_2}")
```